backend/app/auth/routers.go

```diff
@@ -1,13 +1,29 @@
 package auth
 
 import (
+	"fmt"
+	"github.com/gin-gonic/contrib/sessions"
 	"github.com/osscameroon/yotas/app"
+	"os"
 )
 
 func AuthRouter() {
 	router := app.GetApiRouter()
 
-	router.POST("/login", LoginHandler)
-	router.GET("/github/login", GithubCallbackHandler)
-	router.POST("/github/login", GithubCallbackHandler)
+	if secretKey == "" {
+		sk, err := generateSecretKey()
+		if err != nil {
+			_, err := fmt.Fprintln(os.Stderr, err)
+			if err != nil {
+				return
+			}
+			os.Exit(1)
```

Comment:
as we are building an api, I am not keen on having it completely exit if something goes wrong

Reply:
Updated !